config.js

```diff
@@ -13,7 +24,7 @@ var config = {
   apiKeys: (process.env.TTC_APIKEYS || 'true') === 'true',
 
   // Directories in which to run git-standup on for a list of your recent commits.
-  repos: (process.env.TTC_REPOS || '~/Code').split(','),
+  repos: repos.split(','),
```

Comment:
I think we could simplyfy this and make it more robust, also for other platforms like windows:

```suggestion
  repos: (process.env.TTC_REPOS || '~/Code').split(',').map(p => expandHomeDir(p)),
```